source/wrangling.md

```diff
@@ -754,7 +581,7 @@ lang_home_tidy
 ```
```

Comment:
Just above this: `lang_home_tidy.columns = [....]` is not explained at all. What is going on there? Do we need to do that any time we pivot? 

(also if we're renaming columns, we taught students how to do that in Ch 1 -- we should follow that example for consistency)

Reply:
`reset_index` is also not explained at all

Reply:
added the text below. See what you think!